### DISCUSSION TITLE
Card view for the /blog page

### DISCUSSION THREAD

**Discussion by christianavi** [retypeapp/retype#547] [Ideas]
Would be really great if we had an option to have the `/blog` page in a card view instead of lists.

**Comment by geoffreymcgill** [comment]
yes, agreed. We would like to implement some card component options for the `/blog` and elsewhere. 

One small tip for the `/blog` page, you can add a `/blog/readme.md` file and Retype will add that content to the `/blog` page above the blog posts.

The following demonstrates the scenario:

#### `/blog/readme.md`

```md
This is some content that gets added to the `/blog` summary page.
```

The file structure would look like the following:

<img width="372" alt="Screen Shot 2023-05-16 at 9 06 54 PM" src="https://github.com/retypeapp/retype/assets/62210/74d680e9-10f4-4583-b50f-8b91c3f39ba9">

The final rendered `/blog` page should look like the the following:

<img width="836" alt="Screen Shot 2023-05-16 at 9 06 42 PM" src="https://github.com/retypeapp/retype/assets/62210/5e9455e4-c566-4a87-8b9e-e37d4578db57">

Hope this helps.

**Reply by christianavi** [reply]
The `readme.md` on `/blog` will do for now, but I am really looking forward to that card option. Thanks for acknowledging the request by the way.
